testproto2/proto2.letmegrpc.go

```diff
@@ -652,6 +653,34 @@ func (this *htmlProto2) Produce(w net_http.ResponseWriter, req *net_http.Request
 	jsonString := req.FormValue("json")
 	someValue := false
 	msg := &Album{}
+	validateMap := make(map[string]interface{})
+	err := encoding_json.Unmarshal([]byte(jsonString), &validateMap)
+	if err != nil {
+		log.Printf("validator: %s ", err.Error())
+	}
+	if err == nil {
+		for k, v := range validateMap {
+			switch v.(type) {
+			case string:
+				vInt, err := strconv.ParseInt(v.(string), 10, 64)
+				if err != nil {
+					continue
+				}
+				validateMap[k] = vInt
+			case float64:
+				vInt := int(v.(float64))
```

Comment:
Do we have a test case for both of the cases?

Reply:
Oh, I forget to cover the new case !